src/d3a/models/strategy/update_frequency.py

```diff
@@ -108,15 +109,17 @@ class OfferUpdateFrequencyMixin:
 
     def __init__(self,
                  initial_rate_option,
+                 initial_selling_rate,
                  energy_rate_decrease_option,
-                 energy_rate_decrease_per_update
+                 energy_rate_decrease_per_update,
                  ):
         self.initial_rate_option = InitialRateOptions(initial_rate_option)
         self.energy_rate_decrease_option = RateDecreaseOption(energy_rate_decrease_option)
         self.energy_rate_decrease_per_update = energy_rate_decrease_per_update
         self._decrease_price_timepoint_s = {}  # type: Dict[Time, float]
         self._decrease_price_every_nr_s = 0
         self.min_selling_rate = 0
+        self.initial_selling_rate = initial_selling_rate
```

Comment:
I have 2 observations here. First is that there is no final_selling_rate. This implies that the final rate is going to be 0 ? What's the outcome of that, because it was not clear from the story description. 
Second, even if you configure the initial_selling_rate, the risk factor is used for interpolation. Is this actually correct behavior, or should these 2 be decoupled? I haven't checked the energy_rate_decrease_per_update, to see if it also uses the risk parameter (hope not).

Reply:
- 1st: We did not discuss this. IMO it can be an implicit 0, @sarahhambridge what do you think?
- 2nd: IMO they should not be decoupled the initial_selling_rate is only providing the starting point, and should not influence the price decrease. To reinsure you: const_energy_rate_decrease_per_update does not use the risk parameter

Reply:
Hi, it seems to me that the final selling rate should be the min_selling_rate

Reply:
@hannesdiedrich I agree, renaming it to final_selling_rate makes sense.

Reply:
Ok, will implement it.